### PR TITLE
initial fix for FIPS page

### PR DESCRIPTION
**Pull request by matthewhelmke**
## Type of change

Documentation update/fix

### What should this PR do?

Edits/replaces the content on the FIPS page

### Why are we making this change?

As time has passed, much of the information became outdated. This is a stage 1 fix to bring the page into parity with https://www.chainguard.dev/legal/fips-commitment so that we aren't sending mixed messages. The plan is to continue to evolve this page as time permits.

### What are the acceptance criteria? 

Is the information accurate? I'm working with @xnox on this, so when he's happy we will merge.

### How should this PR be tested?

n/a

### CODE REVIEW

**Comment by matthewhelmke** [comment]
This issue will close https://github.com/chainguard-dev/internal/issues/4268

**Review by xnox — APPROVED** [review]
💯 yes

File: content/chainguard/chainguard-images/working-with-images/fips-images.md [hunk 1]
```diff
@@ -18,46 +18,35 @@ weight: 050
...
 
-## What is FIPS?
+‍Chainguard warranties the following with respect to Chainguard container images:
 
-The [Federal Information Processing Standards](https://www.nist.gov/itl/publications-0/federal-information-processing-standards-fips), or FIPS, is a set of standards vendors must adhere to when providing data processing services to the US and Canadian governments. FIPS was developed by the National Institute of Standards and Technology (NIST), an agency within the United States Department of Commerce. 
+Chainguard’s FIPS Images available to be delivered in compliance with FIPS specifications are listed here (each a “Chainguard FIPS Image”). Images will be made available in compliance with FIPS specifications provided a customer’s applicable order form designates the purchase of Chainguard FIPS images.
```
Comment (SharpRake) [inline content/chainguard/chainguard-images/working-with-images/fips-images.md:27]:
```suggestion
Chainguard’s FIPS Images available to be delivered in compliance with FIPS specifications are listed here (each a "Chainguard FIPS Image"). Images will be made available in compliance with FIPS specifications provided a customer’s applicable order form designates the purchase of Chainguard FIPS images.
```

fixing smart quotes.

Also, is there supposed to be a link on "here" in the first sentence?

File: content/chainguard/chainguard-images/working-with-images/fips-images.md [hunk 2]
```diff
@@ -18,46 +18,35 @@ weight: 050
...
 
-NIST has published a number of different FIPS standards. The FIPS standard that concerns Chainguard Images is the 140 series which specify requirements for cryptographic modules. Currently, there are two active FIPS 140 standards: FIPS 140-2 and 140-3. FIPS 140-2 is scheduled to expire on September 21, 2026 as the industry transitions to the newer 140-3 standard. 
+The Chainguard FIPS images contain FIPS-validated software cryptographic modules. Entropy must be provided as specified in its cryptographic policy. The cryptographic module may provide non-approved algorithms, which will result in operating in FIPS non-approved mode. The cryptographic FIPS modules currently provided are:
 
-## Chainguard FIPS Images
+‍- OpenSSL FIPS Provider (CMVP [#4282](https://csrc.nist.gov/projects/cryptographic-module-validation-program/certificate/4282))
```
Comment (SharpRake) [inline content/chainguard/chainguard-images/working-with-images/fips-images.md:31]: This bullet isn't rendering correctly (in the preview anyway). not sure what's going on

File: content/chainguard/chainguard-images/working-with-images/fips-images.md [hunk 3]
```diff
@@ -18,46 +18,35 @@ weight: 050
...
 
-By including this redistribution of the OpenSSL FIPS module, Chainguard is able to provide Images that use FIPS-validated cryptography until the official certificate comes through. You can find the submission status on [NIST’s Modules in Process list](https://csrc.nist.gov/Projects/cryptographic-module-validation-program/modules-in-process/Modules-In-Process-List). We also run tests on each of our FIPS Images to ensure the given application is configured with FIPS-validated cryptography end-to-end. Note that Chainguard's FIPS Images use these modules by default, so there aren't any build flags users need to enable to apply this feature.
+You can find more at: [https://images.chainguard.dev/?category=fips](https://images.chainguard.dev/?category=fips).
 
-Additionally, Chainguard is already prepared for the migration to FIPS 140-3 in the near future because we provide the OpenSSL 3.1 module required for certification. It is expected that the certification process for this module will begin in 2024.
+All of Chainguard's FIPS Images have [STIGs](https://edu.chainguard.dev/chainguard/chainguard-images/working-with-images/image-stigs/).
```
Comment (SharpRake) [inline content/chainguard/chainguard-images/working-with-images/fips-images.md:45]:
```suggestion
All of Chainguard's FIPS Images have [STIGs](/chainguard/chainguard-images/working-with-images/image-stigs/).
```

File: content/chainguard/chainguard-images/working-with-images/fips-images.md [hunk 4]
```diff
@@ -18,46 +18,35 @@ weight: 050
...
-
-You will need to pay attention to how you deploy your Chainguard Images. For example, sometimes people configure installations via Helm in a way that copies an application from an image and deploys it, which would mean that you cannot ensure the code or configuration are unchanged and could put you into a state of non-compliance.
+‍Chainguard will take commercially reasonable efforts to ensure applications utilize FIPS-validated cryptographic modules for any cryptographic operations, provided that the parties acknowledge and agree that certain behaviors or functionalities within such applications, which are beyond the direct control of Chainguard, may not fully adhere to FIPS requirements. In the event there are common vulnerabilities and exposures identified, the Chainguard SLA will apply.
```
Comment (SharpRake) [inline content/chainguard/chainguard-images/working-with-images/fips-images.md:47]:
```suggestion
‍Chainguard will take reasonable efforts to ensure applications utilize FIPS-validated cryptographic modules for any cryptographic operations, provided that the parties acknowledge and agree that certain behaviors or functionalities within such applications, which are beyond the direct control of Chainguard, may not fully adhere to FIPS requirements. In the event there are common vulnerabilities and exposures identified, the Chainguard SLA will apply.
```

"commercially reasonable" makes us sound a little too self-interested, ignore this suggestion if you think it's important though

**Review by SharpRake — APPROVED** [review]
I left a few minor suggestions, lmk if you have questions!

File: content/chainguard/chainguard-images/working-with-images/fips-images.md (hunk 1 above)
Comment (matthewhelmke) [reply content/chainguard/chainguard-images/working-with-images/fips-images.md:27]:
```suggestion
Chainguard’s FIPS Images available to be delivered in compliance with FIPS specifications are listed [here](https://images.chainguard.dev/?category=fips)  (each a “Chainguard FIPS Image”). Images will be made available in compliance with FIPS specifications provided a customer’s applicable order form designates the purchase of Chainguard FIPS images.
```

File: content/chainguard/chainguard-images/working-with-images/fips-images.md (hunk 4 above)
Comment (matthewhelmke) [reply content/chainguard/chainguard-images/working-with-images/fips-images.md:47]: Yeah, this one is a direct quote from the legal page and has legal meaning.

**Pull request merged**